docs/communicate-with-javascript.md

```diff
@@ -3029,3 +3029,77 @@ type person =
 The accessors `nameGet` and `ageGet` will still be generated, but not the
 constructor `person`. This is useful when binding to JavaScript objects while
 preventing any Melange code from creating values of such type.
+
+## Use Melange code from JavaScript
+
+As mentioned in the [build system
+section](build-system.md#commonjs-or-es6-modules), Melange allows to produce
+both CommonJS and ES6 modules. In any of both cases, there is nothing special to
+do to use your Melange-generated JavaScript code from any hand-written
```

Comment:
instead of "nothing special to do" can we mention "it works as expected"?

Reply:
https://github.com/melange-re/melange-re.github.io/pull/57/commits/945a891bc6b9e67661b34fd5c6cd165d07677b0b